packages/ui/src/components/widget/WidgetLabel.js

```diff
@@ -27,6 +29,9 @@ const Addon = styled.span`
     background: ${props => props.theme.label.addon.background};
     color: ${props => props.theme.label.addon.color};
     ${props => props.theme.label.addon.extend.trim()};
+    svg {
```

Comment:
Maybe it's better to let the icon define its style instead of relying on context, what do you think?

Reply:
I agree that styling the svg element from this level is a bit odd. I'll look for a more generic way of achieving this that doesn't assume the child component is an icon...